eslint.config.js

```diff
@@ -0,0 +1,13 @@
+import { addExtensions, litConfig, setDirectoryConfigs, testingConfig } from 'eslint-config-brightspace';
+
+export default [
+	{ ignores: ['.vdiff/'] },
```

Comment:
Wondering if this should also go into the base `litConfig`...

Reply:
I'll make a PR for this and the change above.